prime-router/src/test/kotlin/azure/ActionHistoryTests.kt

```diff
@@ -28,26 +34,24 @@ import java.time.OffsetDateTime
 import java.util.UUID
 import kotlin.test.Test
 import kotlin.test.assertEquals
```

Comment:
Is `assertEquals` still being used? Please convert and remove this ref.

Reply:
Agree.